style.css

```diff
@@ -0,0 +1,139 @@
+/* Universal styles */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+}
+
+.parent-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.centered-div {
+  width: 375vw;
+  height: 100vh;
+}
+
+/* Styling the Navigation bar */
+.nav-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav-container li {
+  list-style: none;
+  margin: 8px 8px 8px 8px;
+  justify-content: left;
+}
+
+nav .logo {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  margin-top: 4px;
+  margin-left: 12px;
+  letter-spacing: normal;
+  text-align: center;
+  padding: 10px 12px;
+}
+
+.menu-bars {
+  color: #7f8cff;
+  margin-right: 24px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+nav .menu-bar {
+  width: 20px;
+  background: #7f8cff;
+  border: 2px solid #7f8cff;
+}
+
+/* Styling the background image */
+main {
+  background-image: url("images/header-shapes_mobile\ \(1\).jpg");
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+/* Styling the main page, home page */
+.content-container {
+  margin-top: 114px;
+  margin-right: 24px;
+  margin-left: 24px;
+  font-family: "Poppins", sans-serif;
+  letter-spacing: normal;
+  text-align: left;
+}
```

Comment:
- [ ] Kindly make sure to align the headline contents using the CSS `flexbox` because it gives us a lot of options to align all our elements, plus, it's responsive.